backend/src/main.py

```diff
@@ -190,6 +190,7 @@ def create_source_node_graph_url(uri, db_name, userName, password, source_url, m
            job_status = "Completed"
            return create_api_response(job_status,data='Unknown URL')
     except Exception as e:
+        source_type = ''
```

Comment:
why would we empty out the source type?

Reply:
@jexp remove empty source_type and handle the exception.